snapshot_tests/update.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+from config import parsers
+from snapshot_tests.snapshot_test import get_snapshot_result_path
+
+
+def generate_snapshot(parser, canteen):
+    url = get_canteen_url(parser, canteen)
+    parsers[parser].parse(url, canteen, 'full.xml')
+
+    result = parsers[parser].parse('', canteen, 'full.xml')
+
+    snapshot_result_path = get_snapshot_result_path(parser, canteen)
+    os.makedirs(os.path.dirname(snapshot_result_path), exist_ok=True)
+    with open(snapshot_result_path, 'w', encoding='utf-8') as result_file:
+        result_file.write(result)
+
+    print("Updated snapshots for {}/{}.".format(parser, canteen))
+
+
+def get_canteen_url(parser, canteen):
+    parser = parsers[parser].sources[canteen]
+    return parser.args[0]
+
+
+def main():
```

Comment:
I think we should integrate this into the `parse.py` (and maybe rename it to just parse?) interface, so that we have a single entry point:
```
./parse city canteen feed
./parse --snapshot city canteen feed
./parse -s city canteen feed
./parse --compare-snapshot city canteen feed
./parse -c city canteen feed
./parse --compare-all-snapshots
./parse compare-all-snapshots (subcommand style? Note that python argparse doesn't seem to be able to handle this…)
```

I will see if I can come up with a simple parser.

Reply:
I don't understand `parse.py` enough to help with this. But the idea sounds great. :)